src/components/MyParty/MyPartyList.jsx

```diff
@@ -2,72 +2,46 @@ import PropTypes from 'prop-types';
 import { Box } from '@mui/material';
 import MyPartySummary from './MyPartySummary';
 
-const MyPartyList = ({ myParties }) => {
-  const handleClickParty = partyId => {
-    console.log(partyId);
-  };
+const MyPartyList = ({ parties }) => {
+  const handleClickParty = partyId => {};
 
   return (
     <Box>
-      {myParties.map(party => {
-        if (party.isLeader) {
-          const {
-            partyId,
-            ottId,
-            ottName,
-            startDate,
-            endDate,
-            isLeader,
-            monthlyReimbursement,
-          } = party;
-
-          return (
-            <MyPartySummary
-              key={partyId}
-              partyId={partyId}
-              ottId={ottId}
-              ottName={ottName}
-              startDate={startDate}
-              endDate={endDate}
-              isLeader={isLeader}
-              monthlyReimbursement={monthlyReimbursement}
-              onClickParty={handleClickParty}
-            />
-          );
-        } else {
-          const {
-            partyId,
-            ottId,
-            ottName,
-            startDate,
-            endDate,
-            isLeader,
-            monthlyFee,
-            totalFee,
-          } = party;
-
-          return (
-            <MyPartySummary
-              key={partyId}
-              partyId={partyId}
-              ottId={ottId}
-              ottName={ottName}
-              startDate={startDate}
-              endDate={endDate}
-              isLeader={isLeader}
-              monthlyFee={monthlyFee}
-              totalFee={totalFee}
-              onClickParty={handleClickParty}
-            />
-          );
-        }
-      })}
+      {parties.map(
+        ({
+          partyId,
+          ottId,
+          ottName,
+          startDate,
+          endDate,
+          isLeader,
+          monthlyReimbursement,
+          remainingReimbursement,
+          monthlyPrice,
+          totalPrice,
+        }) => (
+          <MyPartySummary
+            key={partyId}
+            partyId={partyId}
+            ottId={ottId}
+            ottName={ottName}
+            startDate={startDate}
+            endDate={endDate}
+            isLeader={isLeader}
+            monthlyReimbursement={monthlyReimbursement}
+            remainingReimbursement={remainingReimbursement}
+            monthlyPrice={monthlyPrice}
+            totalPrice={totalPrice}
+            onClickParty={handleClickParty}
+          />
+        ),
```

Comment:
```suggestion
      {parties.map(
        ({ partyId, ...props}) => (
          <MyPartySummary
            key={partyId}
            partyId={partyId}
           {...props}
            onClickParty={handleClickParty}
          />
        ),
```
바로 넘기기만 하는 거라면 요렇게 처리하면 가독성 좋아질것 같아요!

Reply:
도르 저 너무 무지성으로 코드를 짜고 있었네요..
고마워요!!! 이부분 수정해서 반영하겠습니다 👍